srtcore/core.h

```diff
@@ -729,14 +741,16 @@ class CUDT
     //   always increased by one in this call, otherwise it will be increased by the number of blocks
     //   scheduled for sending.
 
+    //int32_t m_iLastDecSeq;                       // Sequence number sent last decrease occurs (actually part of FileCC, formerly CUDTCC)
     int32_t m_iSndLastAck2;                      // Last ACK2 sent back
```

Comment:
m_iLastDecSeq to be deleted

Reply:
Should be gone already, maybe not everything was merged?